packages/web-components/fast-foundation/src/divider/README.md

```diff
@@ -7,25 +7,40 @@ custom_edit_url: https://github.com/microsoft/fast/edit/master/packages/web-comp
 
 A web component implementation of a [horizontal rule](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/hr).
```

Comment:
No change from you here, but while we're updating should add some context in terms of API? For example, it's technically an hr but we also provide an API to set a role of presentation which effectively removes those semantics altogether.

Reply:
My preference is to get this merged and release the docs, since what we have published now is completely broken for our latest release. Then, I think we should establish an ideal component doc template. Something that includes things like properties, attributes, slots, events, examples, referenced design tokens, etc. Once we have the template, I think we should have the individual contributors who worked on each component go submit PRs to update the related docs. In this way we can have a consistent approach but also split up the work. I can tell you that these basic corrections took a lot of time to do across the whole set. I don't think any one person has the bandwidth to do all the ideal updates right now.